playbooks/merge-playbook/library/merge_viya_deployment_files.py

```diff
@@ -60,7 +60,7 @@
 PCP_PORT_LINE    = "      - PCP_PORT"
 PGPOOL_PORT_LINE_BEFORE = "PGPOOL_PORT:"
 POOL_NUMBER_LINE = "        POOL_NUMBER: '0'\n"
-
+PGPOOL_HEARTBEAT_LINE = "        HA_PGPOOL_HEARTBEAT_PORT: ''\n"
```

Comment:
minor nit, but also please add back blank line between these constants and the comments block

Reply:
That's an intentional blank space to address the indentation of the yaml file.

Reply:
actually, @erharb is talking about the original blank line _after_ these constants and before the subsequent comment lines.   adding blank line back there keeps consistency with prior code.  your diff should really just reflect a + line and not any blank line being removed.    I agree with @erharb on this one.

Reply:
I wasn't referring to the space inside the string, just the empty line that used to be there between the constants and comments block

Reply:
Okay.